docs/maui/views/MediaElement.md

```diff
@@ -151,6 +204,29 @@ By default, the media that is defined by the `Source` property doesn't immediate
 
 Platform provided media playback controls are enabled by default, and can be disabled by setting the `ShouldShowPlaybackControls` property to `false`.
 
+### Using MetaData
+
+A `MediaElement` can use metadata for `MediaElement.MetaDataTitle`, `MediaElement.MetaDataArtist` and `MediaElement.MetaDataArtworkUrl` You can set
+the title or artist to show what is currently playing on lockscreen controls for Windows, Mac Catalyst, IOS, and Android. You can set a local or 
+remote URL with artwork for the lockscreen. It should be at least 1080P for best quality to be displayed. It cannot be a local file, or resource. It
```

Comment:
I'm a little confused here @ne0rrmatrix . On the text, you mention that you can set a local or remote url for Artwork, but then you said after: It cannot be. local file or resource. 

So, can we or no use a local file PNG resource as artwork? 😃

Reply:
I can phrase that better. It has to be a local url or remote url. That is http*://* What I mean is it can't be file:// or embed: When I was working
on this a month ago I asked about this. You indicated adding support for URL's was sufficient for now and we could add to it later.

Reply:
Sounds great! Thanks James!